### PR TITLE
fix(mobile): no media viewer

### PR DESCRIPTION
**Pull request by hanszhoou**
## Description

Closes # (NO_ISSUE)
https://mask.atlassian.net/browse/MF-185
https://mask.atlassian.net/browse/MM-205

## Type of change

- [ ] Documentation
- [ ] Code refactoring (Restructuring existing code w/o changing its observable behavior)
- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (a fix or feature that would make something no longer possible to do/require old user must upgrade their Mask Network to this new version)

## Checklist

- [ ] My code follows the style guidelines of this project.
- [ ] I have performed a self-review of my own code.
  - [ ] I have removed all in development `console.log`s
  - [ ] I have removed all commented code.
- [ ] I have commented on my code, particularly in hard-to-understand areas.
- [ ] I have read [Internationalization Guide](https://github.com/DimensionDev/Maskbook/blob/develop/docs/i18n-guide.md) and moved text fields to the i18n JSON file.

If this PR depends on external APIs:

- [ ] I have configured those APIs with CORS headers to let extension requests get passed. 
  - chrome extension: `chrome-extension://[id]`
  - firefox extension: `moz-extension://[id]`
- [ ] I have delegated all web requests to the background service via the internal RPC bridge.

### CODE REVIEW

File: packages/mask/src/plugins/Collectible/SNSAdaptor/ArticleTab.tsx [hunk 1]
```diff
@@ -53,18 +57,12 @@ export function ArticleTab(props: ArticleTabProps) {
 
     return useMemo(() => {
         if (!asset.value) return null
-        const resourceUrl = asset.value.animation_url || asset.value.image_url
+        const resourceUrl = asset.value.image_url || asset.value.animation_url
```
Comment (Lanttcat) [inline packages/mask/src/plugins/Collectible/SNSAdaptor/ArticleTab.tsx]: Is this a business necessity?

Comment (UncleBill) [inline packages/mask/src/plugins/Collectible/SNSAdaptor/ArticleTab.tsx]: Perfer `image_url` over `animation_url` ?

**Comment by hanszhoou** [comment]
@yixing @UncleBill Fixed, if on mobile prefer image over other , that is a business necessity.

**Review by UncleBill — DISMISSED** [review]

**Review by guanbinrui — DISMISSED** [review]

**Review by Jack-Works — DISMISSED** [review]

**Comment by hanszhoou** [comment]
Confirmed by @cythb

**Review by cythb — APPROVED** [review]

**Review by Lanttcat — APPROVED** [review]

**Pull request merged**